docs/getting-started/start-from-scratch-with-empty-db/quickstart-prisma-migrate.md

```diff
@@ -69,6 +69,11 @@ Follow these steps for an initial Prisma setup:
     npx prisma2 migrate save --name 'init' --experimental
     npx prisma2 migrate up --experimental
     ```
+1. Generate the Prisma Client with your updated datamodel::
+   ```
+   npx prisma2 generate --experimental
```

Comment:
```suggestion
   npx prisma2 generate
```

Reply:
I had a feeling this wasn't necessary. I should have realized this command doesn't fall into the `Prisma Migrate` features. Thanks @janpio ! 😊